practice-one/src/styles/components/service.css

```diff
@@ -24,5 +29,22 @@
 }
 
 .service-description-container {
-  margin-left: 20px;
+  margin-left: 50px;
+}
+
+.service-list-wrapper {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(3, 24px);
```

Comment:
Do we need to define rows?

Reply:
Dont need to define it, thanks for your comment